pages/index.js

```diff
@@ -1,7 +1,29 @@
 import Head from "next/head";
 import Link from "next/link";
+import { useState, useEffect } from "react";
 
 export default function About() {
+  const [apiToken, setApiToken] = useState("");
+  const [hasToken, setHasToken] = useState(false);
```

Comment:
Are both of these pieces of state necessary? Can we derive `hasToken` from whether or not `apiToken` is an empty string?

Reply:
good shout. fixed.